src/FileConventions/Library.fs

```diff
@@ -169,7 +198,14 @@ let SplitIntoWords(text: string) =
             if paragraph.Type = CodeBlock then
                 Seq.singleton paragraph
             else
-                let lines = paragraph.Text.Split Environment.NewLine
+                let separatorEolCount = 1
+
+                let lines =
+                    SplitByEOLs
+                        paragraph.Text
+                        separatorEolCount
+                        StringSplitOptions.None
```

Comment:
@aarani are you sure RemoveEmptyEntries doesnt work here???

Reply:
no I'm not sure.

Reply:
RemoveEmptyEntries by nature changes the input which I don't think is the intended behaviour here. Only place I want to use RemoveEmptyEntires is to separate header and body from the commit msg

Reply:
it changes the input?? sorry, what???

Reply:
@aarani ping

Reply:
> it changes the input?? sorry, what???

what RemoveEmptyEntries does is that it removes empty strings in the split result, since we are joining them back together removing empty entires changes the output result which we don't want here.